docs/Reference/web3js-eea-Methods.md

```diff
@@ -113,6 +113,8 @@ Private transaction receipt
 
 Signs and sends a RLP-encoded private transaction to Pantheon using [`eea_sendRawTransaction`](Pantheon-API-Methods.md#eea_sendrawtransaction). 
 
+`sendRawTransaction` support [EEA-compliant privacy](../Privacy/How-To/EEA-Compliant.md) using `privateFor` or [Pantheon-extended privacy](../Privacy/How-To/Pantheon-Privacy.md) using `privacyGroupId`. 
```

Comment:
should "sendRawTransaction" be "eea_sendRawTransaction"? Also, "supports".

I think it reads better with a comma before "or".